src/material/sidenav/drawer.ts

```diff
@@ -276,9 +282,7 @@ export class MatDrawer implements AfterContentInit, AfterContentChecked, OnDestr
           this._elementFocusedBeforeDrawerWasOpened = this._doc.activeElement as HTMLElement;
         }
 
-        if (this._isFocusTrapEnabled && this._focusTrap) {
-          this._trapFocus();
-        }
+        this._trapFocus();
```

Comment:
can we rename this method to `_takeFocus` or `_moveFocus` to reflect the fact that it's not necissarily trapping it? (and update the comment on the method to explain when it's trapped and when it isn't)